package.json

```diff
@@ -8,7 +8,7 @@
     "build": "next build",
     "start": "NODE_ENV=production node server.js",
     "test": "echo \"Error: no test specified\" && exit 1",
-    "prettier-src": "prettier --no-semi --write '{pages,src}/**/*.{js,jsx}'"
+    "prettier-src": "prettier --write '{pages,src}/**/*.{js,jsx}'"
```

Comment:
Flag `--no-semi` is not required since it is already specified in `.prettierrc` as `semi: false`.